src/test/java/unrc/dose/UserStatTest.java

```diff
@@ -0,0 +1,66 @@
+package unrc.dose;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertNotNull;
+
+import org.javalite.activejdbc.Base;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+public class UserStatTest {
+	
+	private static final Logger log = LoggerFactory.getLogger(UserStatTest.class);
+	
+	@BeforeClass
+	public static void beforeAll() {
+		log.info("UserStatTest BeforeClass");
+		Base.open();
+	}
+	
+	@AfterClass
+	public static void afterAll() {
+		log.info("UserStatTest AfterClass");
+		Base.close();
+	}
```

Comment:
The way logging is used is not really beneficial, logging should be implemented in such a way that in case one wants to enabled it while testing, the code should only have something like `log,enable(DEBUG)`.